test/testdata/goldens/embedding/embedding_test/logfile is written with --verbose and on unexpected exceptions.txt

```diff
@@ -305,7 +305,7 @@ $ tool/test-bin/pub_command_runner.dart pub fail
 [E] package:pub/src/command.dart $LINE:$COL   PubCommand.run
 [E] package:args/command_runner.dart $LINE:$COL   CommandRunner.runCommand
 [E]  tool/test-bin/pub_command_runner.dart $LINE:$COL  Runner.runCommand
-[E]  tool/test-bin/pub_command_runner.dart $LINE:$COL  Runner.run
+[E]  tool/test-bin/pub_command_runner.dart $LINE:$COL   Runner.run
```

Comment:
Is this intentional?

Reply:
Yeah - seems the stack-trace got more indented...